src/reducers/todosReducer.ts

```diff
@@ -27,6 +33,13 @@ const todosReducer = (state = initialState, action: TodoActionTypes): ITodoState
         data: [],
         error: action.payload
       };
+    case REMOVE_TODO: {
+      const updatedTodosRemove = state.data.filter((todo) => todo.id !== action.payload);
```

Comment:
as we discussed, this part could be moved to the saga layer to all data management workers